tests/integration/device_test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+include './tests/unit/utils.sh'
+include './tests/integration/utils.sh'
+
```

Comment:
I'm unsure if this test adds value because we are testing just one small part of the device, similar to what we do with the unit test. Consider dropping this test for now or make it like this:
1. In the container, get all the info that `kw device` should return, but manually.
2. run `kw device`
3. Compare the expected values with the ones that you manually collected.

Reply:
Ok, this is a better approach.